libsql/src/database.rs

```diff
@@ -320,6 +331,15 @@ cfg_remote! {
     }
 }
 
+#[cfg(all(feature = "encryption-at-rest", feature = "replication"))]
```

Comment:
Can this be moved to the sys crate? Seems weird to have ffi defs in this crate and in this file as well?

Reply:
yeah, and it should generally be part of bindings.rs, I'll hack it properly in a follow-up, and meanwhile move it to libsql-ffi